go/stellar/stellarsvc/remote_mock_test.go

```diff
@@ -645,6 +697,61 @@ func (r *BackendMock) AssertBalance(accountID stellar1.AccountID, amount string)
 	require.Equal(r.T, amount, r.accounts[accountID].balance.Amount, "account balance")
 }
 
+func (r *BackendMock) GetAccountDisplayCurrency(ctx context.Context, tc *TestContext, accountID stellar1.AccountID) (string, error) {
+	return "USD", nil
+}
+
+func (r *BackendMock) ExchangeRate(ctx context.Context, tc *TestContext, currency string) (stellar1.OutsideExchangeRate, error) {
+	return stellar1.OutsideExchangeRate{
+		Currency: stellar1.OutsideCurrencyCode(currency),
+		Rate:     "0.318328",
+	}, nil
+}
+
+type txDetailsT struct {
+	tx     xdr.Transaction
+	txID   stellar1.TransactionID
+	from   stellar1.AccountID
+	to     stellar1.AccountID
+	amount string
+	asset  stellar1.Asset
+}
+
+func txDetails(txEnvelopeB64 string) (res txDetailsT, err error) {
```

Comment:
Is this used? Might be a merge remnant. I recently replace it with `extract` below.

Reply:
i had a really nasty rebase in this file...i'll remove it.